.github/CODEOWNERS

```diff
@@ -86,7 +89,11 @@
 
 /addons/repair/ @odoo/rd-logistics
 
+/addons/sale/ @odoo/rd-sale
```

Comment:
Note, this will shadow some of the generic rules in this module, in particular the mail_template_data wildcard from @odoo/rd-sm 
Need to check with them if it's ok, or if these file in the sales module should have both ownership.

Reply:
Need ownership on template files as people keep péting them.

Reply:
I wonder then if we shouldn't have the file specific wildcards after all the modules then. I think that might be better to consider those file specific rules more important than anything else by default.